File: src/artesgc/scripts/java/control_flow.java
package artesgc.scripts.java;

public class control_flow {
    public static void main(String[] args){
        boolean value1 = 1==1 && 2!=3,
                value2 = 2!=2 || 3==4;

        if (value1){
            System.out.println("value1 = 1==1 && 2!=3 -> " + true);
        }
        else{
            System.out.println("value1 = 1==1 && 2!=3 -> " + false);
        }

        if (!value2){
            System.out.println("value2 = 2!=2 || 3==4 -> " + false);
        }
        else{
            System.out.println("value2 = 2!=2 || 3==4 -> " + true);
        }

        if (value1 != value2){
            System.out.println("value1 != value2 -> " + true);
        }
        else{
            System.out.println("value1 != value2 -> " + false);
        }
    }
}

File: src/artesgc/scripts/kotlin/control_flow.kt
package artesgc.scripts.kotlin

fun main() {
    val value1 = 1 == 1 && 2 != 3
    val value2 = 2 != 2 || 3 == 4

    if (value1) {
        println("value1 = 1==1 && 2!=3 -> true")
    } else {
        println("value1 = 1==1 && 2!=3 -> false")
    }

    if (!value2) {
        println("value2 = 2!=2 || 3==4 -> false")
    } else {
        println("value2 = 2!=2 || 3==4 -> true")
    }

    if (value1 != value2) {
        println("value1 != value2 -> true")
    } else {
        println("value1 != value2 -> false")
    }
}
